File: src/gestorAplicacion/Personas/Capitanes.java
package gestorAplicacion.Personas;
import gestorAplicacion.Embarcaciones.*;
public abstract class Capitanes {
	private String apodo;
	public Barco barco;
	public int liderazgo;
	public int elocuencia;
	public Tripulacion tripulacion;
	
	public Capitanes(String apodo, int liderazgo, int elocuencia){
		this.apodo=apodo;
		this.elocuencia=elocuencia;
		this.liderazgo=liderazgo;
		
	}
	
	public abstract void yells();
	
	public void vincularBarco(Barco barco) {
		this.barco=barco;
	}
	public void vincularTripulacion(Tripulacion tripulacion) {
		this.tripulacion=tripulacion;
	}
	
	
}

File: practica-2-grupo-1-equipo-7-Barcos/src/gestorAplicacion/personas/capitanes.py
from abc import ABC, abstractmethod

class Capitanes(ABC):
    
    def __init__(self,apodo,liderazgo,elocuencia):
        self._apodo=apodo
        self.liderazgo=liderazgo
        self._elocuencia=elocuencia
        
        
    @abstractmethod
    def yells(self):
        pass
    
    def vincularBarco(self,barco):
        self._barco=barco
    
    def vincularTripulacion(self, tripulacion):
        self._tripulacion=tripulacion